### ISSUE TITLE
Could not find a command named "version".

### ISSUE THREAD

**Issue by panchalAlpit** [flutter/flutter#76374]
I have used "flutter version v1.20.2" and getting the "Could not find a command named "version" " error.

![Screenshot 2021-02-19 at 3 35 51 PM](https://user-images.githubusercontent.com/48007084/108491009-64cbc180-72c9-11eb-97f1-84d9ea0e36e8.png)

Thanks

**Comment by iapicca** [comment]
@panchalAlpit 
did you mean to run `flutter --version` or are you trying to downgrade to an obsolete stable?
if the latter I recommend this 3rd party package [fvm](https://pub.dev/packages/fvm)

**Comment by panchalAlpit** [comment]
@iapicca Thanks for the response.
The "flutter --version" command is working fine, but I want to change the beta version, and when I change the flutter version that time I'm getting the error. I have checked the [doc](https://flutter.dev/docs/reference/flutter-clil) and the "version" command is not there. so how I can change the specific beta version.

**Comment by iapicca** [comment]
@panchalAlpit 
to downgrade try this `stackoverflow` [answer](https://stackoverflow.com/a/61080538/9836706)

**Comment by panchalAlpit** [comment]
I already try this but is not working

**Comment by iapicca** [comment]
@panchalAlpit 
could you post the logs for `flutter downgrade -v`?

**Comment by TahaTesser** [comment]
Hi @panchalAlpit 
Please downgrade using the downgrade command instead, `flutter version` is [deprecated](https://github.com/flutter/flutter/issues/53364), no longer exist as an option

There is an open issue to make this easier to check specific version https://github.com/flutter/flutter/issues/14230
Closing, as this isn't an issue with Flutter itself. If you disagree, please write in the comments and I will reopen it.
Thank you

**Comment by panchalAlpit** [comment]
@TahaTesser I have again shared the image. And my current channel is beta. then I'm going to the specific version of the beta channel. So getting the "Could not find a command named "version" ". after that I'm trying to downgrade the beta version but I haven't any beta version. So what can I do. Please help me for fixing this issue.

Thanks.
![Screenshot 2021-02-19 at 5 46 54 PM](https://user-images.githubusercontent.com/48007084/108503531-8aad9200-72da-11eb-96bb-99f4a21cb8f9.png)

**Comment by TahaTesser** [comment]
Hi @panchalAlpit 
You can't use `flutter downgrade` to a version between `beta` and `stable` unless it was your tool history, the message is expected.

You can install a specific beta version from [releases](https://flutter.dev/docs/development/tools/sdk/releases) or from GitHub [releases](https://github.com/flutter/flutter/tags)

**Comment by agilob** [comment]
I installed flutter on a new PC.

I can't upgrade my flutter project to Flutter 2 due to dependencies.

All other PCs are on 1.22.6.

`flutter version` has been removed, so I need to copy my flutter from Linux to Mac to have previously released version?  Flutter and Dart already have bigger dependency hell problem than npm or java ever had and you're making it a lot worse with this change.

This sucks big time guys.
